internal/hammer/testdata/test_root_ca_private_key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
The Go x509 library uses "TESTING KEY" instead of `PRIVATE KEY`, and they uses a small function to replace this in the code. Could we use this? I quite like that! Example:  https://cs.opensource.google/go/go/+/master:src/crypto/x509/pem_decrypt_test.go;l=83?q=TESTING%20KEY&ss=go%2Fgo:src%2Fcrypto%2Fx509%2F

Reply:
Sure. I've added `TEST` in between.

Reply:
The small function will be added in CT hammer binary.

Reply:
I don't know for sure, but I can see how automatic scanners would alert on a string match on "PRIVATE KEY". Let's see how we get on, we can always change it later.